app/models/record/event_record.rb

```diff
@@ -31,7 +31,7 @@ def convert_xml_to_hash
 
   def parse_single_event_from_xml(event)
     event_data = {
-      id: event.at_xpath("E_ID").try(:text),
+      external_id: event.at_xpath("E_ID").try(:text),
```

Comment:
was passiert wenn das neue Attribute `:external_id` über den JSON2Graphql Converter an den MainAppServer an die GraphQL API gesendet wird? Gibt es dort eine entsprechende Repräsentation?

Wenn nein, bitte **nicht** "einfach" die :external_id im MainServer zu den Models direkt hinzufügen.

Reply:
Dies ist kein neues Attribut wurde in diesem Converter schon vor meinem Urlaub mit Master gemergt.
https://github.com/ikuseiGmbH/smart-village-app-converter-xml2json/blob/master/app/models/record/poi_record.rb
https://github.com/ikuseiGmbH/smart-village-app-converter-xml2json/blob/master/app/models/record/event_record.rb